### PR TITLE
[GT-172] Remove unknown character in edit downtime

### PR DESCRIPTION
**Pull request by Sae126V**

### CODE REVIEW

File: htdocs/web_portal/views/downtime/edit_downtime.php
Comment (rowan04) [inline htdocs/web_portal/views/downtime/edit_downtime.php:230]:
```suggestion
         * just the time values, time widget didn't like timestamp with date)
```
the bracket is never closed :)

Comment (Sae126V) [reply htdocs/web_portal/views/downtime/edit_downtime.php]: Ah right!. 👍

**Review by rowan04 — APPROVED** [review]
lgtm

**Comment by rowan04** [comment]
Wait for confirmation from Greg before merging though, as he was the one that suggested that this pr be made

**Comment by gregcorbett** [comment]
It's weird that GitHub no longer seems to detect the, old, unknown character at all. But hey ho.

**Pull request merged**
